networking/hub-regionA.json

```diff
@@ -891,6 +891,28 @@
                                         "destinationPorts": [
                                             "443"
                                         ]
+                                    },
+                                    {
+                                        "ruleType": "ApplicationRule",
+                                        "name": "LR-NGSA-rule",
```

Comment:
The name could be more general. It allows communication to all apps using the `dev.cse.ms` suffix like grafana, prometheus, etc. And it's not just from LR to ngsa, i only added that initially for the soak testing.

Reply:
ditto